src/main/java/core/basesyntax/service/impl/ParsingStringToFruitTransactionService.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.Parsing;
+
+public class ParsingStringToFruitTransactionService implements Parsing {
+    private static final String CANT_WRITE_EXCEPTION_MESSAGE = "can't parse this line";
+    private static final String SPLITER = ",";
+    private static final int INDEX_OF_OPERATION = 0;
+    private static final int INDEX_OF_FRUIT_NAME = 1;
+    private static final int INDEX_OF_QUANTITY = 2;
+
+    @Override
+    public FruitTransaction parse(String line) {
+        String[] splitedLine = line.split(SPLITER);
+        for (FruitTransaction.Operation operation : FruitTransaction.Operation.values()) {
+            if (operation.getCode().equals(splitedLine[INDEX_OF_OPERATION])) {
+                return new FruitTransaction(operation,
+                        splitedLine[INDEX_OF_FRUIT_NAME],
+                        Integer.parseInt(splitedLine[INDEX_OF_QUANTITY]));
+            }
+        }
```

Comment:
most of this logic involves getting Operation instance from a string, such an action should be inside Operation enum, in `getByCode()` method or smth